api/proto/disperser/v2/README.md

```diff
@@ -0,0 +1,14 @@
+# Should I use the v2 API?
+
+No. The v2 API is currently under active development, and is not yet deployed to any environment.
+
+# Is the v2 API stable?
+
+No, although it is becoming more stable as the v2 design matures.
+
+# What is "v2"?
+
+The EigenDA v2 Architecture is a fundamental redesign of the disperser, and by extension systems that interact
+with the disperser. The v2 Architecture is an important stepping stone towards reducing centralization and for
```

Comment:
nit: The v2 architecture improves robustness, efficiency, and compatibility with upcoming features such as permissionless disperser and data availability sampling.

Reply:
wording updated